www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -1,6 +1,6 @@
 <model>
     <mount>//OPNsense/Nginx</mount>
-    <version>1.35</version>
+    <version>1.35.1</version>
```

Comment:
one of the downsides of trying to align the plugin version with the model version and a hotfix that requires changing the model version without really changing the plugin version ;)